tuxbuild/mainline.tux.yml

```diff
@@ -6,13 +6,4 @@
 sets:
   - name: foo-bar
     builds:
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-nightly, kconfig: multi_v5_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-nightly, kconfig: aspeed_g5_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-nightly, kconfig: multi_v7_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm64, toolchain: clang-nightly, kconfig: defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: x86_64, toolchain: clang-nightly, kconfig: defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-11, kconfig: multi_v5_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-11, kconfig: aspeed_g5_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-11, kconfig: multi_v7_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm64, toolchain: clang-11, kconfig: defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: x86_64, toolchain: clang-11, kconfig: defconfig, make_variables: {LLVM: "1"}}
+      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: i386, toolchain: clang-nightly, kconfig: defconfig, make_variables: {LLVM: "1"}}
```

Comment:
This diff doesn't look right?

Reply:
oh, yeah, I shouldn't have committed this...will undo, tomorrow

Reply:
(I was playing around with the file and invoking tuxsuite locally)

Reply:
done PTAL